ml-proto/spec/check.ml

```diff
@@ -134,9 +134,9 @@ let rec check_expr c et e =
     check_expr_opt c (label c x) eo e.at
 
   | Br_if (x, eo, e) ->
+    check_expr_opt c et eo e.at;
```

Comment:
@lukewagner, I kinda lost track of which PR(s) you are referring to right now, but I suppose the answer is: yes, it would or could. :)